daemon/defaults/defaults.go

```diff
@@ -19,7 +19,7 @@ const (
 	RuntimePath = "/var/run/cilium"
 
 	// RuntimePathRights are the default access rights of the RuntimePath directory
-	RuntimePathRights = 0770
+	RuntimePathRights = 0775
```

Comment:
Changing the directory group ownership sounds much better to me than making the entire directory world readable.